wdio.conf.js

```diff
@@ -0,0 +1,182 @@
+// Stop node from complaining about fake memory leaks at higher concurrency
+require("events").defaultMaxListeners = 20;
+require("dotenv").config();
+
+const path = require("path");
+const uuid = require("uuid/v4");
```

Comment:
`uuid` doesn't show up in package.json and i thought it wasn't part of node's stdlib